lib/main/atom/views/outline/semanticViewPane.ts

```diff
@@ -0,0 +1,74 @@
+import {CompositeDisposable} from "atom"
+import {SemanticView} from "./semanticView"
+import {Disposable} from "atom"
+
+class SemanticViewPane {
```

Comment:
:art: `SemanticViewPane` isn't really a `Pane`, is it? Looks more like `SemanticViewController` at this point. Maybe a good idea to rename it to reflect its function.

Reply:
OK (changed in next commit)